ci/taskcluster/bin/decision.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+import os
+
+import requests
+from taskcluster import fromNow
+from taskcluster.utils import stableSlugId, dumpJson
+
+tasks = [
+    {
+        'name': 'recipe-client-addon:test',
+        'description': 'Test recipe-client-addon with gecko-dev',
+        'command': 'normandy/recipe-client-addon/bin/tc/test.sh',
+        'env': {
+            'GECKO_DEV_URL': 'https://github.com/mozilla/gecko-dev',
+        },
+    },
+    {
+        'name': 'recipe-client-addon:make-xpi',
+        'description': 'Build XPI for recipe-client-addon',
+        'command': 'normandy/recipe-client-addon/bin/tc/make-xpi.sh',
+        'dependencies': ['recipe-client-addon:test'],
+    },
+]
+
+
+def main():
+    _idMaker = stableSlugId()
+
+    def idMaker(name):
+        return _idMaker(name).decode()
+
+    decisionTaskId = os.environ['TASK_ID']
+
+    with requests.Session() as session:
+        for task in tasks:
+            dependencies = [idMaker(name) for name in task.get('dependencies', [])]
+            dependencies.append(decisionTaskId)
+
+            env = task.get('env', {})
+            for key, val in os.environ.items():
+                if key.startswith('GITHUB_'):
+                    env.setdefault(key, val)
+
+            task_id = idMaker(task['name'])
+            res = session.put(f'http://taskcluster/queue/v1/task/{task_id}', data=dumpJson({
+                'metadata': {
+                    'name': task['name'],
+                    'description': task['description'],
+                    'owner': "mcooper@mozilla.com",
```

Comment:
This should actually pull the email of the user that triggered the build, like in `.taskcluster.yml`. I'll fix that.